.circleci/config.yml

```diff
@@ -15,9 +15,48 @@ jobs:
           key: yarn-packages-{{ checksum "yarn.lock" }}
           paths:
             - ~/.cache/yarn
+      - persist_to_workspace:
+          root: /home/circleci
+          paths:
+            - project
```

Comment:
Not sure if there's a more conventional way of saying "persist the whole workspace".